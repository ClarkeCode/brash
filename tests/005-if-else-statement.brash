var num = 25.6

if (num > 20) {
	print "True 1"
}
else {
	print "False 1"
}

if num > 99 {
	print "True 2"
}
else {
	print "False 2"
}
