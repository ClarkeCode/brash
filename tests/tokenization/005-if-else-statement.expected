True 1
False 2
